test/test_jsonchecker.py

```diff
@@ -247,3 +250,27 @@ def test_pass03(self):
             b'{"JSON Test Pattern pass3":{"The outermost value":"must be '
             b'an object or array.","In this test":"It is an object."}}',
         )
+
+
+@pytest.mark.parametrize(
```

Comment:
I know you use the `unittest` syntax above but since `pytest` is used to run, I went with the `pytest` syntax, which I find way more convenient and easy to read.
Are you ok with this?

Reply:
Ok, that's fine. But please move the fail34.json and fail35.json contents to be either inline or in another directory within data because that directory is copied directly from the nativejson-benchmark repo.